core/src/com/unciv/logic/city/managers/CityExpansionManager.kt

```diff
@@ -58,7 +58,13 @@ class CityExpansionManager : IsPartOfGameInfoSerialization {
     fun canBuyTile(tile: Tile): Boolean {
         return when {
             city.isPuppet || city.isBeingRazed -> false
-            tile.getOwner() != null -> false
+            tile.getOwner() != null -> { 
```

Comment:
If X true else false is exactly like just saying "X"

Reply:
Im confused by this

Reply:
"If (<condition>) true else false" is exactly the same as just writing the condition